.editorconfig

```diff
@@ -137,3 +137,6 @@ dotnet_diagnostic.CA2000.severity = none
 
 #Can reassign collections
 dotnet_diagnostic.CA2227.severity = none
+
+#For ConfigureAwait(true) errors when calling async methods
+dotnet_diagnostic.CA2007.severity = none
```

Comment:
Because writing `ConfigureAwait(true)` does nothing. That is the default behaviour for calling an async method. Disabling this warning is appropriate for application code.
Sometimes `ConfigureAwait(false)` can be useful if you don't need to return to the same synchronisation context following the invocation of an async method and result in a small optimisation. We should probably look at where we can use that.